### PR TITLE
remove some unfriendly logs

### PR DESCRIPTION
**Pull request by suyanlong**
修订日志打印的信息。
     如：“set_block ---------------3----------444” 去掉连续的"----"这种符号。

Signed-off-by: suyanlong <592938410@qq.com>

### CODE REVIEW

**Comment by suyanlong** [comment]
`

2017-08-19 14:27:29,430 - INFO - -----cmd_id--------("chain", "new_status")--------------
2017-08-19 14:27:29,430 - INFO - 1 broadcast msg to nodes BROADCAST [0, 2, 3]
2017-08-19 14:27:29,538 - INFO - CHAIN pub status
2017-08-19 14:27:29,538 - INFO - recive msg from origin = 0
2017-08-19 14:27:30,562 - INFO - -----cmd_id--------("consensus", "new_proposal")--------------
2017-08-19 14:27:30,562 - INFO - 1 broadcast msg to nodes BROADCAST [0, 2, 3]
2017-08-19 14:27:37,507 - INFO - CHAIN pub status
2017-08-19 14:27:37,507 - INFO - -----cmd_id--------("chain", "new_status")--------------
2017-08-19 14:27:37,507 - INFO - 1 broadcast msg to nodes BROADCAST [0, 2, 3]
2017-08-19 14:27:37,583 - INFO - CHAIN pub status
2017-08-19 14:27:37,583 - INFO - recive msg from origin = 0
2017-08-19 14:27:45,417 - INFO - CHAIN pub status
2017-08-19 14:27:45,417 - INFO - recive msg from origin = 3
2017-08-19 14:27:45,422 - INFO - CHAIN pub status
2017-08-19 14:27:45,422 - INFO - recive msg from origin = 2
2017-08-19 14:27:45,467 - INFO - CHAIN pub status
2017-08-19 14:27:45,467 - INFO - -----cmd_id--------("chain", "new_status")--------------
2017-08-19 14:27:45,467 - INFO - 1 broadcast msg to nodes BROADCAST [0, 2, 3]
2017-08-19 14:27:45,542 - INFO - CHAIN pub status
2017-08-19 14:27:45,542 - INFO - recive msg from origin = 0

`这是network的打印日志信息

File: chain/forward.rs [hunk 1]
```diff
@@ -308,7 +308,7 @@ pub fn chain_result(chain: Arc<Chain>, rx: &Receiver<(u32, u32, u32, MsgClass)>,
                 trace!("Receive sync {:?} from node-{:?}", BigEndian::read_u64(&content), origin);
                 if let Some(block) = chain.block(BlockId::Number(BigEndian::read_u64(&content))) {
                     let msg = factory::create_msg_ex(submodules::CHAIN, topics::NEW_BLK, communication::MsgType::BLOCK, communication::OperateType::SINGLE, origin, block.write_to_bytes().unwrap());
-                    trace!("-origin-{:?}---chain.blk---{:?}--", origin, communication::OperateType::SINGLE);
+                    trace!("origin {:?}, OperateType {:?}", origin, communication::OperateType::SINGLE);
```
Comment (rink1969) [inline chain/forward.rs]: 这样改了之后，这一条不就跟上一条一样了么？

File: consensus/tdmint/src/core/tendermint.rs [hunk 2]
```diff
@@ -907,7 +907,7 @@ impl TenderMint {
                 self.pre_proc_precommit();
             }
         } else if tminfo.step == Step::PrecommitWait {
-            info!(" ######### height {} round {} PrecommitWait time {:?} ", tminfo.height, tminfo.round, Instant::now() - self.htime);
+            info!(" height {} round {} PrecommitWait time {:?} ", tminfo.height, tminfo.round, Instant::now() - self.htime);
```
Comment (rink1969) [inline consensus/tdmint/src/core/tendermint.rs]: 跟上一条风格不统一，wait到底要不要单独写？

File: chain/forward.rs (hunk 1 above)
Comment (suyanlong) [inline chain/forward.rs]: 一样是什么意思?  我只是把这些“-----”符号去掉，打印日志的时候，太多蚂蚁线了。

File: consensus/tdmint/src/core/tendermint.rs (hunk 2 above)
Comment (suyanlong) [inline consensus/tdmint/src/core/tendermint.rs]: 这种要单独分开吗？

**Pull request merged**